package.json

```diff
@@ -49,7 +50,8 @@
     "@sindresorhus/transliterate": "1.6.0",
     "antd": "4.24.3",
     "axios": "0.27.2",
-    "jsdom": "19.0.0",
+    "cyrillic-to-translit-js": "3.1.0",
```

Comment:
I think that you can update your branch and remove this dependency. We already merged pull request with new translit library

Reply:
done